README.rst

```diff
@@ -20,93 +20,95 @@ How to get started
+	e = create_engine(url)
 	r = e.execute("select 42 from dual").fetchall()
 
 
-The dialect supports two connection urls for create_engine. A DSN (Data Source Name) mode and a host mode:
+The dialect supports two types of connection urls creating an engine. A DSN (Data Source Name) mode and a host mode:
+
+.. list-table::
+    :widths: 50 50
+    :header-rows: 1
 
-========  ====================================================================
-DSN url   'exa+pyodbc://USER:PWD@exa_test'
-Host url  'exa+pyodbc://USER:PWD@192.168.14.227..228:1234/my_schema?parameter'
-========  ====================================================================
+   * - Type
+     - Example
+   * - DSN URL
+     - 'exa+pyodbc://USER:PWD@exa_test'
+   * - HOST URL
+     - 'exa+pyodbc://USER:PWD@192.168.14.227..228:1234/my_schema?parameter'
 
-*Features*:
+Features
+++++++++
 
 - SELECT, INSERT, UPDATE, DELETE statements
 - you can even use the MERGE statement (see unit tests for examples)
 
-*Note*:
+Notes
++++++
 
-- Schema name and parameters are optional for the host url string
+- Schema name and parameters are optional for the host url
 - At least on Linux/Unix systems it has proven valuable to pass 'CONNECTIONLCALL=en_US.UTF-8' as a url parameter. This will make sure that the client process (Python) and the EXASOL driver (UTF-8 internal) know how to interpret code pages correctly.
 - Always use all lower-case identifiers for schema, table and column names. SQLAlchemy treats all lower-case identifiers as case-insensitive, the dialect takes care of transforming the identifier into a case-insensitive representation of the specific database (in case of EXASol this is upper-case as for Oracle)
-- As of EXASol client driver version 4.1.2 you can pass the flag 'INTTYPESINRESULTSIFPOSSIBLE=y' in the connection string (or configure it in your DSN). This will convert DECIMAL data types to Integer-like data types. Creating integers is a factor three faster in Python than creating Decimals.
-
-Troubleshooting
-```````````````
-
-The unixodbc Stack is not the most friendly for programmers. If you get strange errors from the driver mangager, you might have an issue with the names of the unixodbc libs. Have a look at https://github.com/blue-yonder/sqlalchemy_exasol/blob/master/fix_unixodbc_so.sh to find ideas on how to fix this on Ubuntu. Good luck!
+- As of Exasol client driver version 4.1.2 you can pass the flag 'INTTYPESINRESULTSIFPOSSIBLE=y' in the connection string (or configure it in your DSN). This will convert DECIMAL data types to Integer-like data types. Creating integers is a factor three faster in Python than creating Decimals.
 
 Development & Testing
 `````````````````````
 See `developer guide`_
 
 .. _developer guide: https://github.com/exasol/sqlalchemy-exasol/blob/doc/developer_guide/developer_guide.rst
+.. _odbc_driver: https://docs.exasol.com/db/latest/connect_exasol/drivers/odbc/odbc_linux.htm
+.. _test_drive: https://www.exasol.com/test-it-now/cloud/
```

Comment:
Do we want to mention itde or docker-db?

Reply:
Good point was also thinking about itde, but I am unsure if it is user friendly enough just yet (no pypi package etc.).
Regarding the docker-db, I did not think about it, if you would say it is straight forward enough I would vote for us mentioning it (Did not use it on it's own yet).